base_synchro/wizard/base_synchro_view.xml

```diff
@@ -1,54 +1,54 @@
 <?xml version="1.0" encoding="utf-8"?>
 <openerp>
-	<data>
-		<record id="view_base_synchro" model="ir.ui.view">
-			<field name="name">Base Synchronization</field>
-			<field name="model">base.synchro</field>
-			<field name="priority">1</field>
-			<field name="arch" type="xml">
-				<form string="Transfer Data To Server" version="7.0">
-					<group col="4">
-						<field name="server_url" colspan="4" />
-						<field name="user_id" />
-					</group>
-					<footer>
-						<button special="cancel" string="_Close" icon="gtk-cancel" />
```

Comment:
Buttons should have class="oe_highlight" for normal buttons and class="oe_link" for cancel buttons.